### PR TITLE
Forum image upload

### PR DESCRIPTION
**Pull request by Elben85**
1. Allow users to upload **a maximum of 10** Image when creatin / editing a post
2.  The image can only be seen on the ForumPostScreen

### CODE REVIEW

**Review by sugiyem — COMMENTED** [review]
Looks good but I think it would be better if all the removed image are "actually" deleted from firebase storage. If you want to I think I can do it.

**Comment by Elben85** [comment]
> Looks good but I think it would be better if all the removed image are "actually" deleted from firebase storage. If you want to I think I can do it.

I can do it

File: src/components/Forum/ForumPost/RenderPostScreen.js [hunk 1]
```diff
@@ -65,86 +71,169 @@ const RenderPostScreen = ({ renderScreenType }) => {
...
+        () => {},
+        (src) => setPost({ ...post, img: [...post.img, src] }),
+        "Image Added Successfully"
+      )
+    );
+  }
+
+  function handleDeleteImage() {
+    //Users shouldn't know about this background process
+    removeImageFromCloudStorage(post.img[imageIndex]);
```
Comment (sugiyem) [inline src/components/Forum/ForumPost/RenderPostScreen.js:93]: I think the image should be removed from firebase storage when the confirmation button is clicked. In handleDeleteImage, you can store the image-to-be-deleted in some array. All images at this array will then be deleted in handleSubmit.

**Review by sugiyem — APPROVED** [review]
LGTM

**Pull request merged**